library/analytics/src/main/java/org/cru/godtools/analytics/model/AnalyticsActionEvent.kt

```diff
@@ -14,20 +13,13 @@ open class AnalyticsActionEvent(
     locale: Locale? = null,
     systems: Collection<AnalyticsSystem>? = null
 ) : AnalyticsBaseEvent(locale, systems) {
-    companion object {
-        fun String.sanitizeAdobeNameForFirebase(): String = replace(Regex("[ \\-.]"), "_").toLowerCase(Locale.ROOT)
-    }
```

Comment:
we won't be able to remove this until all the tool updates are made, I would leave this alone for now